manifests/plugin/remote_execution/ssh/params.pp

```diff
@@ -0,0 +1,13 @@
+# Remote Execution SSH default parameters
+class foreman_proxy::plugin::remote_execution::ssh::params {
+  include ::foreman_proxy
```

Comment:
No, I think this is fine. Actually surprised that dynflow doesn't need it as well.